src/Accelerators/NNPA/Conversion/ONNXToZHigh/RewriteONNXForZHigh.cpp

```diff
@@ -499,6 +499,10 @@ void getRewriteONNXForZHighDynamicallyLegal(
   // broadcasting.
   addDynamicallyLegalOpFor<ONNXAddOp>(
       target, dimAnalysis, [](ONNXAddOp op, const DimAnalysis *dimAnalysis) {
+        // Check NNPA level.
+        if (!isCompatibleWithNNPALevel(NNPA_Z16)) {
+          return false;
```

Comment:
@mikeessen I am sorry that the logic here is a bit confusing. `false` means the operation is illegal and must be rewritten to run on NNPA, while `true` means the operation is legal and will remain on CPU. So I think we should return true. Same for the other changes in this file.

It's contrary to the logic in `isSuitableForZDNN`. Actually it's like calling `! isSuitableForZDNN()`.

Reply:
Thanks for the clarification.   I updated false to true.